tests/accounts.spec.js

```diff
@@ -0,0 +1,46 @@
+// Jest automatically has babel-jest installed :D
+import '../src/libs/dotenv';
+import { knex, pgp } from '../src/libs/postgres';
+import { insertAccount } from '../src/queries';
+
+let discordId;
+
+beforeEach(async () => {
+  [discordId] = await knex('accounts')
+    .insert({
+      discord_id: '143509445354323968',
```

Comment:
Instead of using a fixed Id, forcing the tests to run in band, I would generate a unique id.